BackwardConfig.cmake

```diff
@@ -133,7 +133,7 @@ find_package_handle_standard_args(Backward
 list(APPEND _BACKWARD_INCLUDE_DIRS ${BACKWARD_INCLUDE_DIR})
```

Comment:
but here its _BACKWARD_INCLUDE_DIRS why would it be different below?

Reply:
I have created a test project to illustrate this issue (backward-cpp used a git submodule):

https://github.com/pkestene/backward-cpp-test

When configure/built with:
cmake -DUSE_BACKWARD_UPSTREAM=ON ..
it fails

When configured/built with
cmake -DUSE_BACKWARD_UPSTREAM=OFF ..
it builds and runs OK.

I really think it is a matter of cmake variable scope; _BACKWARD_INCLUDE_DIRS is not "exported" while BACKWARD_INCLUDE_DIRS is. When the macro is called / expanded _BACKWARD_INCLUDE_DIRS is empty, and the macro fails.